product-service/lambdas/getProductsById.js

```diff
@@ -1,21 +1,67 @@
-import products from '../productList.json' assert {type: 'json'};
-
+import pg from 'pg';
+const { Client } = pg;
 
 const handleResponse = (products = {}, status = 200) => ({
-    headers: {
-      'Content-Type': 'application/json',
-      'Access-Control-Allow-Methods': '*',
-      'Access-Control-Allow-Origin': '*',
-    },
-    statusCode: status,
-    body: JSON.stringify(products),
-  });
-  
+  headers: {
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Methods': '*',
+    'Access-Control-Allow-Origin': '*',
+  },
+  statusCode: status,
+  body: JSON.stringify(products),
+});
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+
+const credentials = {
```

Comment:
Aren't you repeating the same code in each handler? You can easily extract client creation into a separate module

Reply:
Yes, of course. Separation in dedicated module is one of the possible way. In this task, I decided to keep all the code in front of your eyes.